### DISCUSSION TITLE
How to set value for  parameter batch_size

### DISCUSSION THREAD

**Discussion by ignatiev** [Purfview/whisper-standalone-win#360] [Q&A] [answered]
Hi,
I have a GeForce RTX 4080 Super, so I believe this GPU can slightly increase the processing speed. I came across ["Batched Transcription"](https://github.com/SYSTRAN/faster-whisper?tab=readme-ov-file#batched-transcription), but I couldn't find "batch_size" in the parameter list for faster-whisper-xxl.exe.
How can I set the value of  "batch_size" to speed up processing, for example batch_size=8?

**Comment by Purfview** [comment]
This feature is not added yet, as you can see the current standalone is still on revision r194 when the current python repo's revision is 239.
That's because there were various low quality buggy commits merged into the faster-whisper repo, anyway, recently @MahmoudAshraf97 fixed the repo, I think I can release it today updated to the latest.

>I have a GeForce RTX 4080 Super, so I believe this GPU can slightly increase the processing speed.

Batched inference increase speed much more than "slightly", but it decrease accuracy of transcriptions because of the context is not working and it's prone to hallucinations, some areas can be better because VAD does better chunking, but in general there will be some drop in transcription quality.

**Comment by Purfview** [comment, marked as answer]
>How can I set the value of "batch_size" to speed up processing, for example batch_size=8?

Released r239.1 with batched inference,
Just add `--batched` to a command, "batch_size=8" is by default.
